tests/integration_tests/functional/test_cpu_features.py

```diff
@@ -280,9 +303,13 @@ def _test_cpu_rdmsr(context):
     # Baselines are taken by running `msr_reader.sh` on:
     #  * host running kernel 4.14 and guest 4.14 with the `bionic-msrtools` rootfs
     #  * host running kernel 4.14 and guest 5.10 with the `bionic-msrtools` rootfs
-    baseline_df = pd.read_csv(
-        f"../resources/tests/msr/msr_list_{cpu_template}_{utils.get_kernel_version(level=1)}.csv"
-    )
+    #  * host running kernel 5.10 and guest 4.14 with the `bionic-msrtools` rootfs
+    #  * host running kernel 5.10 and guest 5.10 with the `bionic-msrtools` rootfs
+    host_kv = utils.get_kernel_version(level=1)
+    guest_kv = re.search("vmlinux-(.*).bin", context.kernel.name()).group(1)
+    baseline_file_name = f"msr_list_{cpu_template}_{host_kv}host_{guest_kv}guest.csv"
```

Comment:
If the old T2S baseline files are no longer referenced, I suggest deleting them. They will stay in the Git history anyway.

Reply:
I had kept them for future reference but I can delete them